src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/info-card/info-card.component.html

```diff
@@ -0,0 +1,18 @@
+<div [class]="class">
+  <div class="well">
```

Comment:
We can add a class (eg. `class = "info-card"`) and put all scss inside it. Example
```
.info-card{
  .center-block{...
  ...
}
```